Source/Analytics/Web/src/components/Reports/ReportsPerHealthRiskPerRegionLast4Weeks.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from "react";
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import { QueryCoordinator } from "@dolittle/queries";
+import { CaseReportsPerRegionLast7DaysQuery } from "../../Features/CaseReports/CaseReportsPerRegionLast7DaysQuery";
+
+import '../lastWeekTotals/last-week-totals.scss';
+
+class ReportsPerHealthRiskPerRegionLast4Weeks extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            regionsForHealthRisk: [],
+            isLoading: true,
+            isError: false
+        };
+    }
+
+    fetchData() {
+        this.queryCoordinator = new QueryCoordinator();
+        let regionsForHealthRisk = new CaseReportsPerRegionLast7DaysQuery();
+
+        this.queryCoordinator.execute(regionsForHealthRisk).then(queryResult => {
+            if (queryResult.success) {
+                this.setState({
+                    regionsForHealthRisk: queryResult.items.length > 0 ? queryResult.items[0].healthRisks : [],
+                    isLoading: false,
+                    isError: false
+                });
+            }
+            else {
+                this.setState({ isLoading: false, isError: true });
+            }
+        }).catch(_ => {
+            this.setState({ isLoading: false, isError: true });
+        });
+    }
+
+    componentDidMount() {
+        this.fetchData();
+    }
+
+    createRows(regions) {
+        let rows = [];
+        regions.map(region => {
+            let row = Object.keys(region).map(key => (region[key] ? region[key] : 0));
+            let totalReports = region.days0to6 + region.days7to13 + region.days14to20 + region.days21to27;
+            row.push(totalReports);
+
+            rows.push(row);
+        });
+        return rows
+    }
+
+    renderRows() {
+        const currentHealthRisk = this.state.regionsForHealthRisk.find(r => r.healthRiskName == this.props.healthRisk);
```

Comment:
I'm afraid it might be difficult to understand what `this.props.healthRisk` indicates. This is the only way it is mentioned. Maybe a renaming could be smart, such as `selectedHealthRisk`, or some [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html) validation to inform on what this value should be.